docs/en_US/Compression/Tutorial.rst

```diff
@@ -83,30 +83,83 @@ when the value is int type, all quantization types share same bits length. eg.
       quant_bits: 8, # weight or output quantization are all 8 bits
    }
 
+* **quant_dtype** : str or dict of {str : str}
+
+quantization dtype, key is the quantization type, value is the quantization dtype, eg.
+
+.. code-block:: bash
+
+   {
+      quant_dtype: {
+         'weight': 'int',
+         'output': 'uint,
+         },
+   }
+
+when the value is str type, all quantization types share the same dtype. eg.
+
+.. code-block:: bash
+
+   {
+      quant_dtype: 'int', # the dtype of weight and output quantization are all 'int'
+   }
+There are totally two kinds of `quant_dtype` you can set, they are 'int' and 'uint'.
+
+* **quant_scheme** : str or dict of {str : str}
+
+quantization scheme, key is the quantization type, value is the quantization scheme, eg.
+
+.. code-block:: bash
+
+   {
+      quant_scheme: {
+         'weight': 'per_channel_symmetric',
+         'output': 'per_tensor_affine',
+         },
+   }
+
+when the value is str type, all quantization types share the same quant_scheme. eg.
+
+.. code-block:: bash
+
+   {
+      quant_scheme: 'per_channel_symmetric', # the quant_scheme of weight and output quantization are all 'per_channel_symmetric'
```

Comment:
Should we also introduce four quant_scheme and two dtype briefly?

Reply:
done